### ISSUE TITLE
Experimental setup

### ISSUE THREAD

**Issue by ShellyShellyShellyShelly** [saketd403/CROSR#1]
Hi, I would like to ask what is the division of training set, validation set，and open_set of the CIFAR10 dataset. Why my AUROC result is 0.51. which does not reach the 0.71 you gave. Epoch is set to 100.
